apps/server/lib/lexical/server/provider/handlers/go_to_definition.ex

```diff
@@ -1,18 +1,21 @@
 defmodule Lexical.Server.Provider.Handlers.GoToDefinition do
+  alias Lexical.Ast
+  alias Lexical.Document
   alias Lexical.Protocol.Requests.GoToDefinition
   alias Lexical.Protocol.Responses
   alias Lexical.RemoteControl
 
   require Logger
 
   def handle(%GoToDefinition{} = request, env) do
-    case RemoteControl.Api.definition(env.project, request.document, request.position) do
-      {:ok, native_location} ->
-        {:reply, Responses.GoToDefinition.new(request.id, native_location)}
-
+    with {:ok, _document, %Ast.Analysis{valid?: true} = analysis} <-
```

Comment:
This means that when the document can't be analyzed, we can't jump. Do we have any other options?

Reply:
fall back to elixir_sense?

Reply:
Fixed the issue using the `Store.fetch` method.